submissions/jeff-james/scripts/article.js

```diff
@@ -52,12 +53,31 @@ Article.fetchAll = function() {
     // Article.loadAll(//TODO: Process our localStorage!
     // Tip: Be careful when handling different data types between localStorage!
     // );
-    console.log('local storage exists');
-    var localData = localStorage.getItem('data');
-    Article.loadAll(JSON.parse(localData));
-    //TODO: Now call the correct method here that will render the index page.
-    // Article.loadAll();
-    articleView.initIndexPage();
+    $.ajax({
+      url: 'data/hackerIpsum.json',
+      success: function (data, message, xhr) {
```

Comment:
This is a pretty heavy AJAX call when you don't need all the data. Try just getting the headers instead (you can totally do this).